jpa/eclipselink.jpa.test/src/org/eclipse/persistence/testing/tests/jpa/advanced/EntityManagerJUnitTestSuite.java

```diff
@@ -520,15 +525,16 @@ public static Test suiteSpring() {
             tests.add("testNestedFetchQueryHints");
             tests.add("testInheritanceFetchJoinSecondCall");
         }
+        tests.add("testDetachChildObjects");
 
 
         Collections.sort(tests);
         for (String test : tests) {
             suite.addTest(new EntityManagerJUnitTestSuite(test));
         }
-        if (!isJPA10()) {
+        /*if (!isJPA10()) {
             suite.addTest(new EntityManagerJUnitTestSuite("testCascadeDetach"));
-        }
+        }*/
```

Comment:
why is this test commented out?

Reply:
Its was not part of the change , done for some local change . Let me re-build